docs/mitogen.md

```diff
@@ -1,5 +1,7 @@
 # Mitogen
 
+*Warning:* Mitogen support is now deprecated in kubespray due to upstream not releasing an updated version to support ansible 4.x (ansible-base 2.11.x) and above. The CI support has been stripped for mitogen and we are no longer validating any support or regressions for it. The supported playbook and documentation will be removed in a kubespray 2.19.
```

Comment:
Maybe don't be that specific ? `The supported playbook and documentation will be removed in a future kubespray release` for example ? WDYT ?

Reply:
I wanted the message to be loud and clear but yes, it's better to temper it down.

Reply:
I changed the wording, also better English language, but kept the warning note.